playground/src/components/entry-point/entry-point.js

```diff
@@ -28,6 +28,11 @@ export const ApplicationStateMachines = () => (
       />
     )}
     <Route path="/:projectKey/state-machines" component={AsyncStateMachines} />
+    {/* Given a user does not have any projects yet, then the app-shell redirects here. The playground renders a message instead of letting the <RouteCatchAll> handle it. */}
+    <Route
+      path="/account/projects/new"
```

Comment:
Hmm how is this useful? There is no account all running locally

Reply:
If I remember correctly otherwise you and up in a redirect loop as the route catch all kicks in and so forth. Plus it felt kind of okay to have it for debugging as there is no accounts app.

Reply:
Hmm I see. Does it mean that every app needs to do that?

Reply:
Fair question. Something we should check, true. Maybe we should redirect only if it's served by the proxy of something smater.

Reply:
See below.